### DISCUSSION TITLE
Assisting with overall maintenance and up-keep.

### DISCUSSION THREAD

**Discussion by Digitalroot** [Digitalroot/MediaWiki_PHPBB_Auth#48] [General]
@Digitalroot - As the lead developer of [Distributed Proofreaders](https://github.com/DistributedProofreaders/dproofreaders) I have an incentive to keep this plugin alive as we use it at pgdp.net. If you're amenable to me assisting with its overall maintenance and up-keep I'd welcome a conversation about how to make that possible. I can be reached via mail using my github username at the domain kence.org.

_Originally posted by @cpeel in https://github.com/Digitalroot-Technologies/MediaWiki_PHPBB_Auth/issues/46#issuecomment-1022279125_

**Comment by Digitalroot** [comment]
What did you have in mind?

**Reply by cpeel** [reply]
I'm happy to help with issue and PRs as they come up (I wasn't watching the repo before a few days ago -- I should have done that last year). I don't need anything more from you to help with that besides you merging PRs (mine or others) after review and closing issues if we're able to get them resolved. If you wanted to give me enough access to the repo to do that myself, that's fine, but I don't need it.

I would like to see a 4.0 release out, so my question is really to you: how can I help make that happen?

**Comment by cpeel** [comment]
Whenever you feel 4.x is ready for a release, here are some suggested release notes:


# Requirements
* PHP 7.3 or later
* MySQL 5 or later
* MediaWiki 1.31 LTS or later, including 1.35 LTS
* phpBB 3.3
* [PluggableAuth](https://www.mediawiki.org/wiki/Extension:PluggableAuth) MediaWiki extension

This release supports MediaWiki 1.35 and later by using the [PluggableAuth](https://www.mediawiki.org/wiki/Extension:PluggableAuth) MediaWiki extension. After installing PluggableAuth, this extension is a drop-in replacement for prior versions. If your `LoginMessage` or `NoWikiError` messages include HTML, such as links, they will need to be updated to use Wiki markup instead. For instance:
```php
$wgAuth_Config['LoginMessage'] = "'''Please register on the forums to login.''' " .
    "[https://some.domain.com/phpBB/ucp.php?mode=register Click here to create an account.]";
```

This version also includes improved debug messages to troubleshoot configuration and authentication issues. See the [Troubleshooting section](https://github.com/Digitalroot-Technologies/MediaWiki_PHPBB_Auth#troubleshooting) of the `README.md` for more information.
